packages/sandbox/package.json

```diff
@@ -49,6 +49,7 @@
   },
   "devDependencies": {
     "@rollup/plugin-commonjs": "^25.0.7",
+    "@types/debug": "^4.1.12",
```

Comment:
This seems to imply that `sandbox` needs a dep on `debug`; I think that's true given we vendor the vfs into this package?

Reply:
Yes. The DTS build of `@typescript/sandbox` would fail unless `@types/debug` is installed, because `@typescript/vfs` is vendored.

Reply:
Right but I mean that this package should also depend on "debug" too so the code doesn't crash.